src/bootstrap.tsx

```diff
@@ -455,6 +455,20 @@ const parseTheme = (theme?: DefaultApplicationTheme): ThemeProperties => {
   if (theme.secondaryColor) {
     style['--complementaryColor'] = theme.complementaryColor;
   }
+  if (theme.faviconPath) {
```

Comment:
The new key needs to be added to the `Application` model in the [shogun-util](https://github.com/terrestris/shogun-util).

Reply:
https://github.com/terrestris/shogun-util/pull/659